src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -1,14 +1,18 @@
 @import '../../../common/variables';
 
-.bcs-ActivityThread-toggle {
-    margin-left: $sidebarActivityFeedSpacingHorizontal;
-    color: $bdl-box-blue;
-    font-weight: bold;
-
-    &:hover,
-    &:active {
+.bcs-ActivityThread {
+    .bcs-ActivityThread-toggle {
+        margin-top: -9px;
```

Comment:
minor: Do we typically use $bdl-grid-unit for margin units?? I've seen decimal multipliers used in some places, but not sure if this is standard?